src/NServiceBus.Core/Diagnostics/ActivityDecorator.cs

```diff
@@ -15,40 +14,64 @@ public static void PromoteHeadersToTags(Activity activity, Dictionary<string, st
+            { Headers.MessageId, "nservicebus.message_id" },
+            { Headers.CorrelationId, "nservicebus.correlation_id" },
+            { Headers.ReplyToAddress, "nservicebus.reply_to_address" },
+            { Headers.NServiceBusVersion, "nservicebus.version" },
+            //{ Headers.ReturnMessageErrorCodeHeader, "" },
+            { Headers.ControlMessageHeader, "nservicebus.control_message" },
+            { Headers.SagaType, "nservicebus.saga_type" },
+            { Headers.OriginatingSagaId, "nservicebus.originating_saga_id" },
+            { Headers.OriginatingSagaType, "nservicebus.originating_saga_type" },
+            { Headers.DelayedRetries, "nservicebus.delayed_retries" },
+            { Headers.DelayedRetriesTimestamp, "nservicebus.delayed_retries_timestamp" },
+            //{ Headers.ImmediateRetries, "" },
+            //{ Headers.ProcessingStarted, "" },
+            //{ Headers.ProcessingEnded, "" },
+            //{ Headers.TimeSent, "" },
+            { Headers.DeliverAt, "nservicebus.deliver_at" },
+            { Headers.RelatedTo, "nservicebus.related_to" },
+            { Headers.EnclosedMessageTypes, "nservicebus.enclosed_message_types" },
+            { Headers.ContentType, "nservicebus.content_type" },
+            //{ Headers.SubscriptionMessageType, "nservicebus.subscription_message_type" }, // subscription headers are not promoted to tags
+            //{ Headers.SubscriberTransportAddress, "nservicebus.subscriber_address" },
+            //{ Headers.SubscriberEndpoint, "nservicebus.subscriber_endpoint" },
+            { Headers.IsSagaTimeoutMessage, "nservicebus.is_saga_timeout" },
+            { Headers.IsDeferredMessage, "nservicebus.is_deferred" },
+            { Headers.OriginatingEndpoint, "nservicebus.originating_endpoint" },
+            { Headers.OriginatingMachine, "nservicebus.originating_machine" },
+            { Headers.OriginatingHostId, "nservicebus.originating_host_id" },
+            { Headers.ProcessingEndpoint, "nservicebus.processing_endpoint" },
+            { Headers.ProcessingMachine, "nservicebus.processing_machine" },
+            { Headers.HostDisplayName, "nservicebus.host_display_name" },
+            { Headers.HostId, "nservicebus.host_id" },
+            { Headers.HasLicenseExpired, "nservicebus.has_license_expired" },
+            { Headers.OriginatingAddress, "nservicebus.originating_address" },
+            { Headers.ConversationId, "nservicebus.conversation_id" },
+            { Headers.PreviousConversationId, "nservicebus.previous_conversation_id" },
+            { Headers.MessageIntent, "nservicebus.message_intent" },
+            { Headers.TimeToBeReceived, "nservicebus.time_to_be_received" },
+            //{ Headers.DiagnosticsTraceParent, "" },
+            //{ Headers.DiagnosticsTraceState, "" },
+            //{ Headers.DiagnosticsBaggage, "" },
+            { Headers.DataBusContentType, "nservicebus.databus.content_type" },
```

Comment:
This is the only header that has a `.` separator. Was that intentional? Should it be renamed to something like `nservicebus.databus_content_type` for consistency?

Reply:
I kind of like this. It makes databus feel like a separate subcomponent, which feels right.

Reply:
yeah the guidelines suggest hierachical namespaces, so the `.` here is intentional.